scripts/test-1884.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# The purpose of this script is to replicate the failure conditions in GH1884.
+# It attempts to restart tendermint fast enough after first start that
+# the InitChain request gets resent on subsequent service start.
+set -euo pipefail
+set -x
+
+
+sudo systemctl disable --now penumbra tendermint
```

Comment:
Wait, do we have native systemd unit files now?

Reply:
Ha, no, I just use it myself because it's the most convenient option.

Reply:
Filed #1974 to improve the docs a bit. Closing this PR because I don't expect it to be merged; more research required on approach. Will continue discussion in #1884 in subsequent sprints.